apps/web/src/design-system/tabs/tabs.styles.ts

```diff
@@ -0,0 +1,72 @@
+import { createStyles, MantineTheme } from '@mantine/core';
```

Comment:
Please notice that between two tabs should be 30px, currently, there are 50.
I think adding: 
 ```
   tabsList: {
      gap: '30px',
    }, 
```
should do the trick

Reply:
Noted